### PR TITLE
[INLONG-4131][Manager] Add get real-time metrics API

### LINKED ISSUE #4131: [Feature][Manager] Add  group Real-time metrics
### Description

Add  group Real-time metrics in manager-web

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Title Name: [INLONG-4131][Manager] Add inlong Real-time metrics

where *XYZ* should be replaced by the actual issue number.

Fixes #4131 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#
```
Comment (healchow) [inline .github/dependabot.yml:1]: Please do not change this file.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongParser.java [hunk 2]
```diff
@@ -82,6 +83,12 @@ public static WorkflowResult parseWorkflowResult(Response response) {
         return GsonUtil.fromJson(resultData, WorkflowResult.class);
     }
 
+    public static InlongGroupTotalMetricResponse parseGroupHeartbeat(Response response) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongParser.java:86]: Please add a java doc.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/InlongGroupServiceImpl.java [hunk 3]
```diff
@@ -421,4 +434,38 @@ public void saveOrUpdateExt(String groupId, List<InlongGroupExtInfo> infoList) {
         LOGGER.info("success to save or update inlong group ext for groupId={}", groupId);
     }
 
+    @Override
+    public InlongGroupTotalMetricResponse getMetric(InlongGroupMetricRequest request) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/InlongGroupServiceImpl.java:438]: Please refer to the `HeartbeatController`, and re-consider the design of those interfaces.

**Comment by LvJiancheng** [comment]
The previous plan has been changed，so

**Pull request closed without merging**